### PR TITLE
Localized parameters

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Localizes the parameters in available through the unit selection screen as well as the ones in the StartupGUI.
In total adds 142 new stringtable entries.
image
image

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### CODE REVIEW

File: A3A/addons/core/Params.hpp [hunk 1]
```diff
@@ -143,44 +143,44 @@ class Params
...
+        texts[] = {$STR_A3A_Params_generic_none,"20%","40%","60%","80%", $STR_A3A_Params_generic_allowed};
         default = 20;
     };
     class memberDistance
     {
-        title = "Max distance of guests from the closest member or HQ";
+        title = $STR_A3A_Params_memberDistance_title;
         values[] = {4000,5000,6000,7000,8000,10000,16000,-1};  // 16000 is left as backwards compatibility
-        texts[] = {"4 km","5 km","6 km","7 km","8 km","10 km","16 km","Unlimited"};
+        texts[] = {$STR_A3A_Params_generic_4km, $STR_A3A_Params_generic_5km, $STR_A3A_Params_generic_6km, $STR_A3A_Params_generic_7km, $STR_A3A_Params_generic_8km, $STR_A3A_Params_generic_10km, $STR_A3A_Params_generic_16km, $STR_A3A_Params_generic_unlimited};
```
Comment (jaj22) [inline A3A/addons/core/Params.hpp:183]: Do the X km ones really need stringtabling?

Comment (Bob-Murphy) [reply A3A/addons/core/Params.hpp:183]: I don't know if in all languages it's `x km` or maybe also `km x`. This way translators can adjust to whatever needed.
Worst case, we have a hand full of strings more than we actually need.

File: A3A/addons/core/Params.hpp [hunk 2]
```diff
@@ -143,44 +143,44 @@ class Params
...
-        title = "MEMBERSHIP OPTIONS";
+        title = $STR_A3A_Params_TitleMembership_title;
         values[] = {""};
         texts[] = {""};
         default = "";
     };
     class membershipEnabled
     {
-        title = "Enable Server Membership";
+        title = $STR_A3A_Params_TitleMembership_title;
```
Comment (jaj22) [inline A3A/addons/core/Params.hpp:153]: Wrong stringtable entry, copypasted from above.

Comment (Bob-Murphy) [reply A3A/addons/core/Params.hpp]: fixed

File: A3A/addons/core/Stringtable.xml [hunk 3]
```diff
@@ -12301,5 +12301,385 @@
...
+      </Key>
+      <Key ID="STR_A3A_Params_A3A_builderPermissions_engi">
+        <Original>Engineers</Original>
+      </Key>
+      <Key ID="STR_A3A_Params_A3A_builderPermissions_both">
+        <Original>Both</Original>
+      </Key>
+      <Key ID="STR_A3A_Params_TitleMembership_title">
+        <Original>Enable Server Membership</Original>
+      </Key>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml]: Actually looks like this one has the text that's supposed to be in the missing one.

Comment (Bob-Murphy) [reply A3A/addons/core/Stringtable.xml]: fixed with the previous one

File: A3A/addons/core/Params.hpp [hunk 4]
```diff
@@ -2,135 +2,135 @@ class Params
...
-        title = "Initial Rebel Faction Manpower";
+        title = $STR_A3A_Params_initialHr_title;
         values[] = {0, 8, 16, 24, 32, 50};
         texts[] = {"0","8","16","24","32","50"};
         default = 8;
     };
     class A3A_idleTimeout
     {
-        title = "Time before a player is considered AFK";
+        title = $STR_A3A_Params_A3A_idleTimeout_title;
```
Comment (jaj22) [inline A3A/addons/core/Params.hpp:103]: I wouldn't recommend keeping the second A3A tag in these. Will make it confusing and error-prone if the tags are added to vars later.

Comment (Bob-Murphy) [reply A3A/addons/core/Params.hpp]: fixed for core and maps

**Review by jaj22 — APPROVED** [review]
Approved but make sure you run it again after the changes and double-check that everything looks right.

**Comment by Bob-Murphy** [comment]
Re-tested.
Everything works and logs are clean.

**Pull request merged**